qgispluginci/changelog.py

```diff
@@ -1,54 +1,165 @@
-"""Changelog parser.
+#! python3  # noqa E265
```

Comment:
```suggestion
#!/usr/bin/env python3
```
?

Reply:
This is an automatic replacement of my config. I don't exactly remember but the old-fashioned shebang `#!/usr/bin/env` is not recomended anymore, especially on certain distributions and `#! python3` is compatible by the Windows python launcher.

`# noqa E265` is a flag to disable flake8 lint alert

Reply:
ok